.devcontainer/devcontainer.json

```diff
@@ -10,6 +10,7 @@
 			// Options
 			"INSTALL_NODE": "false",
 			"NODE_VERSION": "lts/*",
+			"TEST": "false"
```

Comment:
Please remove this line, you probably pushed it accidentally.

Reply:
Fixed.